sdk/tables/azure-data-tables/azure/data/tables/_authentication.py

```diff
@@ -211,7 +211,7 @@ def on_challenge(self, request: PipelineRequest, response: PipelineResponse) ->
         if self._discover_tenant:
             self.authorize_request(request, scope, tenant_id=challenge.tenant_id)
         else:
-            self.authorize_request(request, scope)
+            self.authorize_request(request, *scope)
```

Comment:
could scope also be a tuple on L212? Or a string on this line? I'm not sure how self._discover_tenant comes into play here

Reply:
`scope` value in line212 should be type `str`

Reply:
discussed offline